apollo-compiler/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'java'
 apply plugin: 'kotlin'
 
-targetCompatibility = JavaVersion.VERSION_1_6
-sourceCompatibility = JavaVersion.VERSION_1_6
+targetCompatibility = JavaVersion.VERSION_1_7
```

Comment:
⚠️  This was needed since composite builds require all sub-projects to have compatible java version. 

This is the only place where I don't feel comfortable. Please let me know if this is an issue.

Reply:
Here is what I got without this change.

![image](https://user-images.githubusercontent.com/763339/61982401-ea5b9280-affc-11e9-8983-514e3cf7f108.png)